Library/Formula/riemann.rb

```diff
@@ -19,16 +17,16 @@ def shim_script
   end
 
   def install
-    if (etc/'riemann.config').exist?
-      (prefix/'etc').install 'etc/riemann.config' => 'riemann.config.guide'
+    if (etc/"riemann.config").exist?
+      (prefix/"etc").install "etc/riemann.config" => "riemann.config.guide"
     else
-      etc.install 'etc/riemann.config'
+      etc.install "etc/riemann.config"
     end
```

Comment:
Can you change this to just use `etc.install`? It's now smart enough that it won't overwrite existing files.